### PR TITLE
Cancel updates

### PR DESCRIPTION
**Pull request by Exr0n**
Fixed `standard/commands/Cancel.java`

### CODE REVIEW

**Comment by Exr0n** [comment]
**Please do not review, as the code changes are not up to standard as defined by Daniel on slack:**

> Hey everyone, a few notes about standard updates:
> for requirements, please use the following format (its a little different than what we were doing before, but this is what Rohan and the WPILib devs do): 
```java
 @Override
  public Set<Subsystem> getRequirements() {
    return Set.of(...requirements...);
  }
```
> `...requirements...` is a Vararg, so just provide a command separated list of subsystems (or a single subsystem with no commas)
> for `initDefaultCommands`, please delete the entire method and add a javadoc to the class that the method was formerly a member of saying what command should be set as the default. Default commands are now set when the subsystem is instantiated as subsystems no longer need to have knowledge of their default commands. If you have questions about javadocs, please ask me or nikhil.
> Generally avoid wildcard imports (*) as they clutter the local namespace. Please only import what you use.
> `command.start()` is now essentially `command.schedule()`. `command.initialize()` is still `command.initialize()`.

**Comment by Exr0n** [comment]
Fixed the issues, please review :)

File: commands/Cancel.java [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
 		return true;
 	}
 
-	@Override
-	protected void end() {}
-
-	@Override
-	protected void interrupted() {}
+	public Set<Subsystem> getRequirements()
+	{
```
Comment (aemino) [inline commands/Cancel.java]: formatter formatter formatter plz

File: commands/Cancel.java [hunk 2]
```diff
@@ -1,35 +1,34 @@
...
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+
+/**
+ * This class cancels a command
+ */
+public class Cancel extends CommandBase {
 	protected final Command command;
 
 	public Cancel(Command command) {
```
Comment (NikhilSuresh24) [inline commands/Cancel.java]: take in a `CommandBase`

File: commands/Cancel.java [hunk 3]
```diff
@@ -1,35 +1,34 @@
...
 	}
 
-	@Override
-	protected void end() {}
-
-	@Override
-	protected void interrupted() {}
+	public Set<Subsystem> getRequirements() {
+		return Set.of();
+	}
```
Comment (NikhilSuresh24) [inline commands/Cancel.java]: delete

File: commands/Cancel.java [hunk 4]
```diff
@@ -1,35 +1,34 @@
...
 		command.cancel();
 	}
 
-	@Override
-	protected void execute() {}
-
-	@Override
-	protected boolean isFinished() {
+	public boolean isFinished() {
 		return true;
```
Comment (NikhilSuresh24) [inline commands/Cancel.java]: can you make it return true only if `!command.isScheduled`

**Pull request merged**